lambda_function.py

```diff
@@ -19,9 +19,13 @@ class BaseHTTPResponse:
 
 def get_entity_info_and_return_slack_message(record: dict) -> str:
     body = json.loads(record["body"])
-    entity_ref: str = body["ref"]
+    new_state_value = body["NewStateValue"]
+    queue = list(filter(lambda x: x['value'] != 'QueueName', body["Trigger"]["Dimensions"]))[0]['value']
+    if new_state_value == "ALARM":
+        slack_message = f"A message was sent to the DLQ for {queue}"
+    else:
+        slack_message = f"The DLQ for {queue} has been cleared"
```

Comment:
Do we need this or could we use something more generic like `Cloudwatch alarm <name> has entered state ALARM`. I don't think we'll be putting much more monitoring around our lambdas but when we come to run code on-prem we're likely to want another alarm configured for no messages received/polled, maybe some monitoring on the box itself (depending where the line between dev/ops falls).
Are we doing a different alarm for each queue as well? It's very likely that multiple queues could send to their DLQs at the same time and we'd want to know that.

Reply:
I'll change it to Cloudwatch alarm <name> has entered state ALARM, that makes sense.
We can have different alarms for different queues but they can all point to this lambda and it will still work.